task/02-numbers-tasks.js

```diff
@@ -143,7 +143,7 @@ function getLastDigit(value) {
  * '-525.5'     => -525.5
  */
 function parseNumberFromString(value) {
-    throw new Error('Not implemented');
+    return new Number(value);
```

Comment:
the function should return number type
Number(value) will simply convert the expression into a number primitive value, while new Number(value) will create a wrapper object around the converted number value
typeof new Number(value) is "object"